src/Microsoft.Identity.Client/Internal/IdToken.cs

```diff
@@ -90,20 +90,19 @@ public static IdToken Parse(string idToken, RequestContext requestContext)
                             idTokenBody = (IdToken) serializer.ReadObject(stream);
                         }
                     }
-                    catch (SerializationException ex)
+                    catch (SerializationException)
```

Comment:
I will remove the exception. it should be bubbled up.

---
In reply to: [109345288](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/282#discussion_r109345288) [](ancestors = 109345288)